app/controllers/calendar_controller.rb

```diff
@@ -74,16 +74,14 @@ def simple_404
 
   def handle_bank_holiday_ics_calendars
     if scope == "bank-holidays"
-      i18n_backend.send(:init_translations) unless i18n_backend.initialized?
-      translations = i18n_backend.send(:translations)
-      division_key = translations[I18n.locale][:common][:nations].key(params[:division]).to_s
-      params[:division] = "common.nations." + division_key
-    end
-  end
+      division_slug = {
```

Comment:
It looks like we're missing the welsh entries for Scotland and Northern Ireland (Yr Alban, Gogledd Iwerddon). Those calendars can't be downloaded. We could update the integration test for all 6 calendars to ensure we have coverage for all of them?